teamcity/flake8_v3_plugin.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from flake8.formatting import base
+
+from teamcity.messages import TeamcityServiceMessages
+from teamcity import __version__
+
+
+class TeamcityReport(base.BaseFormatter):
+    name = 'teamcity-messages'
+    version = __version__
+    messages = TeamcityServiceMessages()
+
+    options_added = False
+
+    @classmethod
+    def add_options(cls, parser):
+        if not cls.options_added:
+            parser.add_option('--teamcity', default=False,
```

Comment:
To have this always run on TeamCity, why not just use `is_running_under_teamcity()` for the default value?